recipes/tbb/bld.bat

```diff
@@ -0,0 +1,22 @@
+rem  @echo off
+
+@rem See https://github.com/wjakob/tbb
+cp "%RECIPE_DIR%\CMakeLists.txt" .
+if errorlevel 1 exit 1
+cp "%RECIPE_DIR%\version_string.ver.in" .
+if errorlevel 1 exit 1
+
+mkdir build
+cd build
+
+set CMAKE_CONFIG="Release"
+
+if not defined CMAKE_GENERATOR EXIT 1
+
+cmake -LAH -G%CMAKE_GENERATOR%                               ^
```

Comment:
but don't you still have to use cmake to generate the makefile for nmake?  Or does tbb ship with a windows makefile?